design/ceph/ceph-nfs-ganesha.md

```diff
@@ -220,9 +220,9 @@ Directory, and FreeIPA.
 Prototype information detailed on Rook blog:
 https://blog.rook.io/prototyping-an-nfs-connection-to-ldap-using-sssd-7c27f624f1a4
 
-NFS-Ganesha (via libraries within its container) is the client to SSSD. As of Ceph v17.2.3, the Ceph
```

Comment:
Let's revert this file, it's telling a specific version where the fix and feature were available.